homeassistant/components/lg_netcast/media_player.py

```diff
@@ -185,7 +185,7 @@ def supported_features(self):
     @property
     def media_image_url(self):
         """URL for obtaining a screen capture."""
-        return self._client.url + "data?target=screen_image"
+        return f"{self._client.url}data?target=screen_image&_={str(int(datetime.now().timestamp()))}"
```

Comment:
Two things:

1. You do not need to cast it into an string when using f"-strings:
```suggestion
        return f"{self._client.url}data?target=screen_image&_={int(datetime.now().timestamp())}"
```
2. Have you tried this without casting it to an int, therefore leaving it as a float?

Reply:
float also works. commited.